backend/app/app/services/chat_agent/meta_agent.py

```diff
@@ -34,8 +34,10 @@ def get_conv_token_buffer_memory(
         ConversationTokenBufferMemory: The ConversationTokenBufferMemory object.
     """
     agent_config = get_agent_config()
+
+    # We use gpt-4 in ConversationTokenBufferMemory to standardize tokenization
     llm = get_llm(
-        agent_config.common.llm,
+        'gpt-4',
```

Comment:
@kaikun213 forcing it to use gpt-4's tokenizer here through tiktoken because otherwise langchain tries to import claude's tokenizer from transformers otherwise (and I don't think we want to add that to dependencies). Does that work?